browser/extensions/api/brave_sync_api.cc

```diff
@@ -51,9 +60,10 @@ ExtensionFunction::ResponseAction BraveSyncSyncSetupErrorFunction::Run() {
       brave_sync::SyncSetupError::Params::Create(*args_));
   EXTENSION_FUNCTION_VALIDATE(params.get());
 
-  BraveSyncService* sync_service = GetBraveSyncService(browser_context());
+  BraveSyncService* sync_service =
+    GetSyncService(browser_context());
```

Comment:
lint: 4 spaces for continuation of a line

Reply:
addressed in 3b1d457dc14d5a36353c45699459d728c5cb08ef